auth-api/tests/unit/api/test_invitation.py

```diff
@@ -0,0 +1,203 @@
+    """Assert that an invitation can be POSTed."""
+    headers = factory_auth_header(jwt=jwt, claims=TEST_JWT_CLAIMS)
+    rv = client.post('/api/v1/users', headers=headers, content_type='application/json')
+    rv = client.post('/api/v1/orgs', data=json.dumps(TEST_ORG_INFO),
+                     headers=headers, content_type='application/json')
+    dictionary = json.loads(rv.data)
+    org_id = dictionary['id']
+    new_invitation = {
+        'recipientEmail': 'test@abc.com',
+        'sentDate': '2019-09-09',
+        'membership': [
+            {
+                'membershipType': 'MEMBER',
+                'orgId': str(org_id)
+            }
+        ]
+    }
+    print(new_invitation)
+    rv = client.post('/api/v1/invitations', data=json.dumps(new_invitation),
+                     headers=headers, content_type='application/json')
+    print(json.loads(rv.data))
+    assert rv.status_code == http_status.HTTP_201_CREATED
+
+
+def test_add_invitation_invalid(client, jwt, session):  # pylint:disable=unused-argument
+    """Assert that POSTing an invalid invitation returns a 400."""
+    headers = factory_auth_header(jwt=jwt, claims=TEST_JWT_CLAIMS)
+    rv = client.post('/api/v1/users', headers=headers, content_type='application/json')
+    rv = client.post('/api/v1/orgs', data=json.dumps(TEST_ORG_INFO),
+                     headers=headers, content_type='application/json')
+    dictionary = json.loads(rv.data)
+    org_id = dictionary['id']
+    new_invitation = {
+        'recipientEmail': 'test@abc.com'
+    }
+    print(new_invitation)
+    rv = client.post('/api/v1/invitations', data=json.dumps(new_invitation),
+                     headers=headers, content_type='application/json')
+    assert rv.status_code == http_status.HTTP_400_BAD_REQUEST
+
+
+def test_get_invitations_by_user(client, jwt, session):  # pylint:disable=unused-argument
+    """Assert that an invitation by a user can be retrieved."""
+    headers = factory_auth_header(jwt=jwt, claims=TEST_JWT_CLAIMS)
+    rv = client.post('/api/v1/users', headers=headers, content_type='application/json')
+    rv = client.post('/api/v1/orgs', data=json.dumps(TEST_ORG_INFO),
+                     headers=headers, content_type='application/json')
+    dictionary = json.loads(rv.data)
+    org_id = dictionary['id']
+    new_invitation = {
+        'recipientEmail': 'test@abc.com',
+        'sentDate': '2019-09-09',
+        'membership': [
+            {
+                'membershipType': 'MEMBER',
+                'orgId': str(org_id)
+            }
+        ]
+    }
+    rv = client.post('/api/v1/invitations', data=json.dumps(new_invitation),
+                     headers=headers, content_type='application/json')
+    rv = client.get('/api/v1/invitations', headers=headers, content_type='application/json')
+    invitation_dict = json.loads(rv.data)
+    assert rv.status_code == http_status.HTTP_200_OK
+    assert invitation_dict['invitations']
```

Comment:
We should probably make sure there is the correct number of invitations returned here (should only be 1 for this test case)

Reply:
Done